src/collections/_documentation/platforms/android/index.md

```diff
@@ -597,3 +597,67 @@ SentryAndroid.init(this,options -> {
 If you want to have control over what breadcrumbs are attached to your data, you can register a hook that is called with each new breadcrumb that's created.
 
 That allows the user to decide whether and how a breadcrumb should be sent.
+
+## Integrating the NDK
+
+To use the Android NDK in your native code, include the Sentry NDK libraries into your project so that the compiler can link the libraries during the build.
+
+To do so, use the AndroidNativeBundle Gradle plugin that copies the native libraries from the Sentry NDK into the location that can provide links via the CmakeLists.txt configuration file.
+
+First, we need to declare the dependency in the project build.gradle file:
+
+```groovy
+dependencies {
+    classpath 'com.android.tools.build:gradle:3.5.2'
```

Comment:
I think we shouldn't have this here. The users need to have gradle anyway and this could be confusing as we're saying they need 3.5.2
```suggestion
```